src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -214,8 +214,11 @@ internal class ConnectionManager : IConnectionManager {
 
         private bool IsValidLocalConnection(string name, string path) {
             try {
-                var info = new RInterpreterInfo(name, path);
-                return info.VerifyInstallation();
+                Uri uri;
+                if (Uri.TryCreate(path, UriKind.Absolute, out uri) && uri.IsFile) {
```

Comment:
So if this method returns `false` for remote connection, `LastActiveConnection` will be set to null and user will be switched from remote connection to the first available local. Is it a desired behavior?